arviz/plots/ppplot.py

```diff
@@ -108,24 +97,31 @@ def plot_pp(
     if backend_kwargs is None:
         backend_kwargs = {}
 
-    if data_pairs is None:
-        data_pairs = {}
-
     datasets = [getattr(data, group) for group in groups]
 
     if var_names is None:
         var_names = list(datasets[0].data_vars)
-    var_names = _var_names(var_names, datasets[0])
-    vars = [var_names]
-    for i in range(1, len(datasets)):
-        var_name =[data_pairs.get(var, var) for var in var_names]
-        var_name = _var_names(var_name,  datasets[i])
+
+    if isinstance(var_names, str):
+        var_names = [var_names]
+
+    if isinstance(var_names, list) or isinstance(var_names, tuple):
+        var_names = [list(var_names) for _ in datasets]
+
+    vars = []
+    for i in range(len(datasets)):
+        var_name = _var_names(var_names[i],  datasets[i])
         vars.append(var_name)
```

Comment:
this should not be needed anymore after the changes above

Reply:
I was doing it earlier to check if the `var_names` are present in all groups or not? In that case, the `_var_names` will throw the error.

Reply:
I missed that, it can still be added once var_names is a list of lists:

    var_names = [_var_names(vars, dataset) for vars, dataset in zip(var_names, datasets)]

or something similar

Reply:
Okay :)